Dockerfile

```diff
@@ -1,24 +1,35 @@
-FROM node:6-stretch
+FROM node:6-stretch AS build
 LABEL maintainer="will.price94@gmail.com"
 LABEL version="0.0.1"
-RUN apt-get update -qq && \
-    apt-get install -y python
 # Prevent npm from spamming
+ENV NPM_CONFIG_LOGLEVEL=warn
```

Comment:
Why? Do the lines below not achieve this, if not can we get rid of them?

Reply:
yeah `npm config set loglevel` didnt seem to work for me (not sure why).  removed it